record_linux/lib/record_linux.dart

```diff
@@ -251,90 +191,83 @@ class RecordLinux extends RecordPlatform {
     }
   }
 
-  // Playback/Loopback:
-  // device #1: FOO (High Definition Audio) - Default
-  // Default Format: 2 channel, 48000 Hz
-  // Capture:
-  // device #1: Microphone (High Definition Audio Device) - Default
-  // Default Format: 2 channel, 44100 Hz
-  Future<List<InputDevice>> _listInputDevices(
-    String recorderId,
-    List<String> out,
-  ) async {
+  List<InputDevice> _parseInputDevices(List<String> output) {
```

Comment:
Add output sample of listing to better understand the parsing

Reply:
Sure thing, by the way currently the id being stored is the Source number which changes on reboot or system logouts. Must look into the dump of this command to see if there is a more permanent device id!